src/providers/inventory-data.ts

```diff
@@ -36,8 +36,9 @@ export class InventoryData {
       .map(this.extractData);
   }
 
-  return(items: Array<Object>) {
-    return Observable.fromPromise(Promise.resolve());
+  return(tag: string) {
+    return this.authHttp.delete(ApiUrl + Links.rental + '/' + tag)
```

Comment:
True. I just created [an issue](https://github.com/emmanuelroussel/stockpile-app/issues/88) to abstract all the api calls from that file. It will be more efficient than doing a few methods at a time in different PRs.